src/hooks/useProposalsSearchParams.ts

```diff
@@ -14,7 +14,7 @@ export type SearchParams = {
   search: string
   searching: boolean
   timeFrame: string
-  order: 'ASC' | 'DESC' | undefined
+  order?: SortingOrder | 'RELEVANCE'
```

Comment:
I think we could benefit from representing the `SortingCriteria` (timeframe, relevance) as something different from the `SortingOrder` (asc, desc) 🙏